sycl/doc/extensions/FilterSelector/FilterSelector.adoc

```diff
@@ -0,0 +1,85 @@
+= SYCL Proposals: Filter Selector
+James Brodman <james.brodman@intel.com>
+v0.1
+:source-highlighter: pygments
+:icons: font
+== Introduction
+This document presents an extension on top of the SYCL specification.  The goal of this extension is to provide a new device selector class that allows expressing common but non-trivial requirements for device selection in a simple manner.
+
+== Filter Selector
+
+The filter selector is a new device selector class that accepts a string of one or more filters that refine the set of devices that may be returned when the selector's `select_device` method is invoked.  Devices that match the specified filter(s) are ranked by the default_selector to determine which device is ultimately selected.
```

Comment:
I don't understand how default_selector fits in here.  Should it be filter_selector?

Reply:
The default selector is used to score devices that match the filter, in case more than one device does.